### PR TITLE
Support Rails 7

### PR DESCRIPTION
**Pull request by tjchambers**
Upgrade to support at least Rails 6 and 7 (current Rails and Ruby versions)

### CODE REVIEW

File: lib/paper_trail_scrapbook/changes.rb [hunk 1]
```diff
@@ -119,9 +123,9 @@ def build_associations
       @build_associations ||=
         Hash[
           klass
-            .reflect_on_all_associations
-            .select { |a| a.macro.equal?(:belongs_to) }
-            .map { |x| [x.foreign_key.to_s, assoc_klass(x.name, x.options)] }
+        .reflect_on_all_associations
```
Comment (dinjas) [inline lib/paper_trail_scrapbook/changes.rb:126]: indentation

File: lib/paper_trail_scrapbook/changes.rb [hunk 2]
```diff
@@ -30,18 +30,18 @@ def initialize(*)
     def change_log
       text =
         changes
-          .map { |k, v| digest(k, v) }
-          .compact
-          .join("\n")
+        .map { |k, v| digest(k, v) }
```
Comment (dinjas) [inline lib/paper_trail_scrapbook/changes.rb:33]: indentation intentional?

**Review by dinjas — APPROVED** [review]
Awesome, looks good to me. I left a couple of comments re: whitespace.

**Comment by dinjas** [comment]
It looks like there was an issue with CI - I missed that.

File: lib/paper_trail_scrapbook/changes.rb (hunk 1 above)
Comment (tjchambers) [reply lib/paper_trail_scrapbook/changes.rb:126]: Per Rubocop this is correct

File: lib/paper_trail_scrapbook/changes.rb (hunk 2 above)
Comment (tjchambers) [reply lib/paper_trail_scrapbook/changes.rb:33]: per rubocop this is correct

File: lib/paper_trail_scrapbook/changes.rb (hunk 1 above)
Comment (dinjas) [reply lib/paper_trail_scrapbook/changes.rb:126]: Rubocop must have some issues

**Pull request merged**